waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -809,10 +809,11 @@ proc validateMessage*(rlnPeer: WakuRLNRelay, msg: WakuMessage,
     waku_rln_invalid_messages_total.inc(labelValues=["invalid_epoch"])
     return MessageValidationResult.Invalid
 
-  if not rlnPeer.validateRoot(msg.proof.merkleRoot):
-    debug "invalid message: provided root does not belong to acceptable window of roots", provided=msg.proof.merkleRoot, validRoots=rlnPeer.validMerkleRoots
-    waku_rln_invalid_messages_total.inc(labelValues=["invalid_root"])
-    return MessageValidationResult.Invalid
+  ## TODO: FIXME
+  # if not rlnPeer.validateRoot(msg.proof.merkleRoot):
+  #   debug "invalid message: provided root does not belong to acceptable window of roots", provided=msg.proof.merkleRoot, validRoots=rlnPeer.validMerkleRoots
+  #   waku_rln_invalid_messages_total.inc(labelValues=["invalid_root"])
```

Comment:
This can still be included

Reply:
Good point, bringing it back

Reply:
Sure, done https://github.com/status-im/nwaku/pull/1246/commits/98e43be71881ea67ff3387932f184fa0ff0b8cdd